certificate.conf

```diff
@@ -0,0 +1,22 @@
+server {
+    listen 443 ssl http2 default_server;
+    listen [::]:443 ssl http2 default_server;
+    include     /etc/nginx/conf.d/self-signed.conf;
+    include     /etc/nginx/conf.d/ssl-params.conf;
+    server_name 127.0.0.1;
+
+    location / {
+        try_files $uri @app;
+    }
+    location @app {
+        include uwsgi_params;
+        uwsgi_pass unix:///tmp/uwsgi.sock;
+    }
+    location /static {
+        alias /app/static;
+    }
+    location /health {
```

Comment:
@linglp This is an elegant solution. Currently this endpoint does not seem to be documented in the OpenAPI description of Schematic. That may be fine if you consider the health endpoint is "private", i.e. should not be exposed to the external users.